yarn-project/cli/src/cmds/add_contract.ts

```diff
@@ -1,4 +1,12 @@
-import { AztecAddress, CompleteAddress, EthAddress, Fr, Point } from '@aztec/aztec.js';
+import {
+  AztecAddress,
+  ContractInstanceWithAddress,
+  EthAddress,
+  Fr,
+  Point,
+  getContractClassFromArtifact,
+} from '@aztec/aztec.js';
+import { computeContractAddressFromInstance, computePublicKeysHash } from '@aztec/circuits.js/contract';
```

Comment:
I see that this PR introduces a circuits.js dependency to cli. Are these required by devs and if so should they be exported from aztec.js instead?

Reply:
I'd expect not, both should be hidden under Contract or Account abstractions. Still, I got a task at the very end to review the APIs, so I can take a look at it then.

Also, note that the transitive dependency already existed.